_posts/2020-03-11-guide-for-unmaintained-open-source-projects.markdown

```diff
@@ -0,0 +1,177 @@
+looking for help maintaining the project.
+
+**Do not** demand an answer from the maintainers. Maintainers don't owe you much,
+so you will be lucky if you get a response to a nice request. So, 
+**please be nice**.
+
+You don't need to offer yourself as a potential maintainer for the project, 
+maybe there are other people out there who are in a better position to 
+take over.
+
+If you don't get any response for a month, then you can move on to the next 
+section.
+
+#### Start your own fork
+
+You could start your own OSS project based on a fork. Before you do that, make 
+sure that you check the contribution graph for the project. There might be 
+someone else updating a fork that already has all the changes you need.
+
+> Fun fact: [Forking a project used to be considered a "bad word"](https://www.bacula.org/why-forking-is-bad/) 
+> but I think the rise of GitHub turned that around. 
+
+This is what [Solidus](https://Solidus.io) did with [Spree](https://spree.com).
+A group of contributors got together and decided to start a new open source, 
+e-commerce framework which started as a fork of Spree.
+
+> The [governance model for Solidus](https://solidus.io/blog/2019/07/10/governance-published.html) 
+> is actually quite interesting and definitely a step in the right direction based 
+> on lessons learned from Spree and Solidus' history
+
+Assuming you didn't skip any steps, you may want to post a comment in the issue 
+you submitted initially. There you can say that until the maintainers respond, 
+you will be maintaining a fork of this project.
+
+### For OSS Maintainers
+
+What can you do as someone who is maintaining an OSS project to make it easier 
+on yourself? What can you do as someone who is done maintaining the project 
+and wants to move on? Here are some ideas based on my own experience and advice 
+from other maintainers.
+
+#### Look for Co-Maintainers
+
+You don't have to do it alone. It's very likely that many of the users and 
+contributors of your project would be happy to step up their commitment to 
+co-maintain it. [You can go so much further if you build a team of co-maintainers](https://opensource.guide/best-practices/#share-the-workload).
+
+Nobody wants to spend their holidays answering comments, reviewing pull requests, 
+or triaging bugs. The nice thing about finding co-maintainers is that they can 
+step up when you can't.
+
+A quick and simple way to do this is to post an issue to your repository. 
+Another one is to post a notice at the beginning of the README. You don't have 
+to accept all the people who are interested, but maybe some recent contributors 
+will be interested.
+
+#### Look for someone to take over the project
+
+If you can't actively maintain the project anymore, you can update your README 
+sharing this with your users and contributors. You can say that your project 
+needs a new maintainer and wait for people to reach out to you.
```

Comment:
Maybe add a section about what could happen if no people reach to you - The transition between looking for a maintainer to "this projects is not maintained" anymore